.travis.yml

```diff
@@ -203,7 +203,19 @@ script:
         make check-integration || exit 1
         make check-themes || exit 1
         do_codecov functionaltests
-        do_codecov_gcov c_code
+
+        # Run gcov.  Upload for codecov directly, gets done later for coveralls.
+        # NOTE: files for the main target have the build dir as prefix, but
+        #       otherwise it is the source directory?!
```

Comment:
> the same directory as the object file is used, so build dir. However, I guess that if the compiler is invoked with a relative path, then the program ends up using a relative path as well...?

The object files in `build//CMakeFiles/awesome.dir` seem to have `build/` prefix, while the files in `build//CMakeFiles/{lgi-check.dir,test-gravity.dir}` do not have it.

> I'm mostly working based on GitHub notifications and so for me PRs can easily "fall through the cracks".

I see.  I'll request your review in case it takes to long.
Also, I will make sure to add WIP preffixed more often.